tests/helpers/pactHelper.js

```diff
@@ -70,7 +70,7 @@ const applicationXmlResponseHeaders = {
   'Content-Type': 'application/xml; charset=utf-8'
 }
 const textPlainResponseHeaders = {
-  'Content-Type': 'text/plain; charset=utf-8'
+  'Content-Type': MatchersV3.regex('text/plain(;()?charset=(utf|UTF)-8)?', 'text/plain; charset=utf-8')
```

Comment:
Can we do loose match here or not acceptable?

Matches:
`text/plain; charset=utf-8`
`text/plain;charset=utf-8`
`text/plain; charset=UTF-8`
`text/plain`

Reply:
I will collect the data